hedyorg/hedy
# [FIX] Issue 3577- Replaced old content under why is Hedy Gradual and Hedy in the news

**Description**

Replaces an old video explaining why is Hedy gradual with a new link to a strange loop talk by Felienne.  Adds a new website under "Hedy in the news".

**Fixes _issue or discussion number_**

Issue 3577

**How to test**
run app.py
Go to http://0.0.0.0:8080/
On the first page it opens, scroll down to the "Why is Hedy Gradual" title and see an embedded YouTube video of a strange
-------
Click on the learn more option in the navigation menu. Scroll down to Hedy in the news section. You should see a new link called Tech Optimism - A solution to the IT problem. Click on the link and it should take you to https://pom.show/2022/08/12/een-oplossing-voor-het-it-tekort/
